app/error/cloud_api_error.py

```diff
@@ -0,0 +1,12 @@
+from fastapi import HTTPException
+
+
+class CloudApiException(HTTPException):
+    """Class that represents a cloud api error"""
```

Comment:
Yeah I've been doubting this choice since I made it. It's basically the same as `HTTPException`. I usually create a base error when writing applications and extend all errors from that. This way e.g. in the trust registry facade we can add a TrustRegistryException, that will also nicely return the correct code to the frontend, but we can also catch is as an CloudApiException.

We could also do this with `HTTPException` directly however. I think just habit. Do you want me toch change it?

Reply:
Ah I see. Well, leave it like this. Don't wanna micro-manage the way you write code. Half of my comment on this was also to make sure I'm not missing sth that's going on and I'm not seeing/understanding.